.dotnet.azure/src/Custom/Chat/AzureChatClient.cs

```diff
@@ -41,15 +37,21 @@ protected AzureChatClient()
     public override AsyncCollectionResult<StreamingChatCompletionUpdate> CompleteChatStreamingAsync(IEnumerable<ChatMessage> messages, ChatCompletionOptions options = null, CancellationToken cancellationToken = default)
     {
         options ??= new();
-        options.StreamOptions = null;
+        if (_apiVersion <= ServiceVersion.V20240701Preview)
```

Comment:
Is this check still correct? Or has stream_options been removed from this version too?

Reply:
Sadly, yes, it's removed. Again.